lib/cli.rb

```diff
@@ -1,11 +1,15 @@
 # frozen_string_literal: true
 
-# Resonsible for any input/output via the command line
+# Responsible for any input/output via the command line
 class CLI
   def display_message(text)
     puts text
   end
 
+  def prompt_user_input
```

Comment:
It seems to me that this method name is misleading

Reply:
Do you think `get_user_input` be clearer?